org.gottcode.FocusWriter.json

```diff
@@ -0,0 +1,35 @@
+{
+	"app-id": "org.gottcode.FocusWriter",
+	"runtime": "org.kde.Platform",
+	"runtime-version": "5.9",
+	"sdk": "org.kde.Sdk",
+	"command": "focuswriter",
+	"rename-icon": "focuswriter",
+	"rename-desktop-file": "focuswriter.desktop",
+	"rename-appdata-file": "focuswriter.appdata.xml",
+	"finish-args": [
+		"--socket=wayland",
+		"--socket=x11",
+		"--filesystem=home",
+		"--share=network"
```

Comment:
Does it really need network access?

Reply:
Without it, this error is shown:

Qt: Session management error: Could not open network socket

But I don't know why it would need network access. I've removed it for now.